core/core-keybindings.el

```diff
@@ -106,24 +106,40 @@ pairs. For example,
 
 (defalias 'evil-leader/set-key 'spacemacs/set-leader-keys)
 
+(defun spacemacs//acceptable-leader-p (key)
```

Comment:
I appreciate the effort :smiley: But I still think "bind-key" should have been named "dear-leader" :joy: